packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -92,12 +105,73 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone {
      * @inheritdoc IFaultDisputeGame
      */
     function step(
-        uint256 _prestateIndex,
+        uint256 _stateIndex,
         uint256 _parentIndex,
```

Comment:
super nit: Would it be better to rename `_parentIndex`, maybe `claimIndex` instead? The naming makes less sense from a defense pov.